### DISCUSSION TITLE
7/28(금) RIBs 알아보기

### DISCUSSION THREAD

**Discussion by jjunhaa0211** [Info-iOS/iOStudy#200] [General]
RIBs를 MVVM보다 먼저 공부하라고 한 이유가 있습니다. 한 10배는 더 어렵기에 RIBs를 해보고 느꼈는데 RIBs보니까 다른것도 좀 쉬워보여서 한번 해보기를 바랍니다. 🌳

**Comment by Eunho0922** [comment]
넵
